config/navigation.yaml

```diff
@@ -1,9 +1,9 @@
 drive:
   map:
-    max_driving_effort: 1.0
-    min_driving_effort: -1.0
-    max_turning_effort: 1.0
-    min_turning_effort: -1.0
+    max_driving_effort: 0.7
+    min_driving_effort: -0.7
+    max_turning_effort: 0.7
+    min_turning_effort: -0.7
```

Comment:
Not sure we want to merge these to master, kinda just wanted to keep from putting too much strain on the system while turning on concrete